src/tests/elementary/meson.build

```diff
@@ -110,6 +109,10 @@ if get_option('ethumb')
   elementary_suite_src += ['elm_test_thumb.c']
 endif
 
+if get_option('dbus')
+  elementary_suite_src += ['elm_test_atspi.c']
```

Comment:
```suggestion
  elementary_suite_src += files('elm_test_atspi.c')
```

Reply:
`elementary_suite_src` don't use files, it is only a list of names